pkg/command/resolve_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command_test
+
+import (
+	"bytes"
+	"path/filepath"
+	"testing"
+
+	"github.com/mattmoor/mink/pkg/bundles"
+	"github.com/mattmoor/mink/pkg/command"
+	"github.com/mattmoor/mink/pkg/tests"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	expectedDigest = "sha256:8e65ec4b80519d869e8d600fdf262c6e8cd3f6c7e8382406d9cb039f352a69bc"
+	verbose        = false
+)
+
+func TestCommandResolve(t *testing.T) {
```

Comment:
I generally like this little pattern.  I'll make it useful to test the other commands 🤩